tools/interop_matrix/client_matrix.py

```diff
@@ -200,7 +200,15 @@ def should_build_docker_interop_image_from_release_tag(lang):
         {
             'v1.6.6': None
         },
-        #{'v1.7.1': None}, Failing tests
+        {
+            'v1.7.2': None
```

Comment:
Please make sure the new versions are uploaded to GCR using create_matrix_images.py before merging. Otherwise the test will break.

Reply:
yes, I just verified that all three of those versions are in the GCR tag list.